transport/http2_server.go

```diff
@@ -845,3 +972,14 @@ func (t *http2Server) RemoteAddr() net.Addr {
 func (t *http2Server) Drain() {
 	t.controlBuf.put(&goAway{})
 }
+
+func getJitter(v time.Duration) time.Duration {
```

Comment:
Optional, random idea if you prefer: func jitter(v time.Duration) time.Duration and do time.Add() to also apply the jitter here.